Profile-observation-nngm-tumorstadium-registration-form.xml

```diff
@@ -126,6 +120,14 @@
       <min value="1" />
       <mustSupport value="true" />
     </element>
+    <element id="Observation.encounter">

```

Comment:
Observation.encounter can't reference EpisodeOfCare, you would need an extension for it. I think we use it already in other places, there should be a core extension for it.